### PR TITLE
Fixes9+10

### LINKED ISSUE #9: md = markdown.Markdown(['full_yaml_metadata']) didn't work for me
Basic usage on pypi invokes extension with 
`md = markdown.Markdown(['full_yaml_metadata'])`

I got the error:
```
>>> md = markdown.Markdown(['full_yaml_metadata'])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: __init__() takes 1 positional argument but 2 were given
```
But this did work (or at least gave another error)
`md = markdown.Markdown(extensions=['full_yaml_metadata'])`

### LINKED ISSUE #10: KeyError: 'configs' on running
After getting extension to load properly, got error:
```
>>> import markdown
>>> md = markdown.Markdown(extensions=['full_yaml_metadata'])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/phil/Share/Projects/full-yaml-test/venv/lib/python3.6/site-packages/markdown/core.py", line 100, in __init__
    configs=kwargs.get('extension_configs', {}))
  File "/home/phil/Share/Projects/full-yaml-test/venv/lib/python3.6/site-packages/markdown/core.py", line 126, in registerExtensions
    ext = self.build_extension(ext, configs.get(ext, {}))
  File "/home/phil/Share/Projects/full-yaml-test/venv/lib/python3.6/site-packages/markdown/core.py", line 181, in build_extension
    return module.makeExtension(**configs)
  File "/home/phil/Share/Projects/full-yaml-test/python-markdown-full-yaml-metadata/full_yaml_metadata.py", line 45, in makeExtension
    return FullYamlMetadataExtension(configs=configs)
  File "/home/phil/Share/Projects/full-yaml-test/venv/lib/python3.6/site-packages/markdown/extensions/__init__.py", line 42, in __init__
    self.setConfigs(kwargs)
  File "/home/phil/Share/Projects/full-yaml-test/venv/lib/python3.6/site-packages/markdown/extensions/__init__.py", line 73, in setConfigs
    self.setConfig(key, value)
  File "/home/phil/Share/Projects/full-yaml-test/venv/lib/python3.6/site-packages/markdown/extensions/__init__.py", line 61, in setConfig
    if isinstance(self.config[key][0], bool):
KeyError: 'configs'
```

I fixed by changing 
```
def makeExtension(configs: dict={}):
    return FullYamlMetadataExtension(configs=configs)
```
to 
```
def makeExtension(*args, **kwargs):
    return FullYamlMetadataExtension(*args, **kwargs)
```
Hope this helps.

### PR DESCRIPTION
**Pull request by philbarker**
change to README.md on how extension is invoked in python interactive shell closes #9
change to  full_yaml_metadata.py to makeExtension parameters closes #10

### CODE REVIEW

**Comment by sivakov512** [comment]
@philbarker, thanks a lot!
Can you also update dependencies versions in[ requirements.txt](https://github.com/sivakov512/python-markdown-full-yaml-metadata/blob/master/requirements.txt#L1) and "fix" version of [Markdown in setup.py](https://github.com/sivakov512/python-markdown-full-yaml-metadata/blob/master/setup.py#L10) accordingly?
This allows us to confirm with tests that everything works as expected.

**Comment by philbarker** [comment]
Will this do? Please forgive me, I'm new at this. Feel free to assume I don't know what I'm doing.

File: setup.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
     author='Nikita Sivakov',
     author_email='cryptomaniac.512@gmail.com',
     description='YAML metadata extension for Python-Markdown',
-    install_requires=['Markdown', 'PyYAML==4.2b4'],
+    install_requires=['Markdown', 'PyYAML>=3.13'],
```
Comment (sivakov512) [inline setup.py]:
```suggestion
    install_requires=['Markdown==3.0.1', 'PyYAML>=3.13'],
```

File: README.md [hunk 2]
```diff
@@ -21,18 +21,18 @@ import markdown
...
-	- Stupid content
+  - Lorem Ipsum
+  - Stupid content
 ...
 
 Lorem Ipsum is simply dummy text.
 """
 
-md = markdown.Markdown(['full_yaml_metadata'])
+md = markdown.Markdown(extensions = ['full_yaml_metadata'])
```
Comment (sivakov512) [inline README.md]: You should update tests also =)

**Comment by sivakov512** [comment]
With the version update we were able to reproduce the error you reported. It remains to update the tests and make sure that everything works correctly.

**Comment by philbarker** [comment]
Sorry. I'm doing something wrong. I don't get errors here, but must be missing something. I'll come back to this later.

**Comment by sivakov512** [comment]
@philbarker, you can look for errors in CI and fix them.

**Comment by philbarker** [comment]
That looks better. I must get Travis working on my own repos.

**Comment by sivakov512** [comment]
Looks good, @philbarker!
Are you test it manually? It works well?

**Comment by philbarker** [comment]
Yes, I think it works.

⁣Get TypeApp for Android ​

**Comment by sivakov512** [comment]
Thank you, @philbarker!
Feel free to try other projects, contribute and star them if you like.

**Pull request merged**